src/main/java/org/htwkvisu/domain/ScoreValue.java

```diff
@@ -0,0 +1,198 @@
+    private BigInteger poiId;
+
+    @Column(name = "osm_id", nullable = false)
+    private BigInteger osmId;
+
+    @Column(name = "point", columnDefinition = "Geometry", nullable = false)
+    @Type(type = "org.htwkvisu.domain.GeometryType")
+    private Point point;
+
+    @Column(name = "name", nullable = false)
+    private String name;
+
+    @Column(name = "aeroway")
+    private String aeroway;
+
+    @Column(name = "bus")
+    private String bus;
+
+    @Column(name = "tram")
+    private String tram;
+
+    @Column(name = "train")
+    private String train;
+
+    @Column(name = "railway")
+    private String railway;
+
+    @Column(name = "amenity")
+    private String amenity;
+
+    @Column(name = "healthcare")
+    private String healthcare;
+
+    @Column(name = "tourism")
+    private String tourism;
+
+    @Column(name = "research_institution")
+    private String researchInstitution;
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getAeroway() {
+        return aeroway;
+    }
+
+    public void setAeroway(String aeroway) {
+        this.aeroway = aeroway;
+    }
+
+    public String getBus() {
+        return bus;
+    }
+
+    public void setBus(String bus) {
+        this.bus = bus;
+    }
+
+    public String getTram() {
+        return tram;
+    }
+
+    public void setTram(String tram) {
+        this.tram = tram;
+    }
+
+    public String getTrain() {
+        return train;
+    }
+
+    public void setTrain(String train) {
+        this.train = train;
+    }
+
+    public String getRailway() {
+        return railway;
+    }
+
+    public void setRailway(String railway) {
+        this.railway = railway;
+    }
+
+    public String getAmenity() {
+        return amenity;
+    }
+
+    public void setAmenity(String amenity) {
+        this.amenity = amenity;
+    }
+
+    public String getHealthcare() {
+        return healthcare;
+    }
+
+    public void setHealthcare(String healthcare) {
+        this.healthcare = healthcare;
+    }
+
+    public String getTourism() {
+        return tourism;
+    }
+
+    public void setTourism(String tourism) {
+        this.tourism = tourism;
+    }
+
+    public String getResearchInstitution() {
+        return researchInstitution;
+    }
+
+    public void setResearchInstitution(String researchInstitution) {
+        this.researchInstitution = researchInstitution;
+    }
+
+    public BigInteger getPoiId() {
+        return poiId;
+    }
+
+    public void setPoiId(BigInteger poiId) {
+        this.poiId = poiId;
+    }
+
+    public BigInteger getOsmId() {
+        return osmId;
+    }
+
+    public void setOsmId(BigInteger osmId) {
+        this.osmId = osmId;
+    }
+
+    public Point getPoint() {
+        return point;
+    }
+
+    public void setPoint(Point point) {
+        this.point = point;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
StringBuilder is faster then simple concatination.
If this function is called often, please replace with StringBuilder.